package.json

```diff
@@ -51,14 +53,14 @@
     "prosemirror-tables": "^0.7.9"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.5.4",
```

Comment:
To do it we need to update the babel version, this is much more work and I think it's not part of that PR, maybe we should step back this and focus only on the `emptyCell` fix, what do you think?

Reply:
I got issues having jest transpiling the sources, but you're right and @d4rkr00t also suggested to put this in another PR... so I've removed this commit from this PR.